RoboticonColony/Assets/Code/Inventory.cs

```diff
@@ -30,23 +29,17 @@ public Inventory(int money, int ore, int power, int roboticons)
         }
 
         Money = money;
-        Tiles = new List<Tile>();
-        Items = new Dictionary<ItemType, int>();
-        Items[ItemType.Ore] = ore;
-        Items[ItemType.Power] = power;
-        Items[ItemType.Roboticon] = roboticons;
+        tiles = new List<Tile>();
+        items = new Dictionary<ItemType, int>();
+        items[ItemType.Ore] = ore;
+        items[ItemType.Power] = power;
+        items[ItemType.Roboticon] = roboticons;
     }
 
     /// <summary>
     /// Creates an empty inventory instance, which holds all resources for a player or market.
     /// </summary>
     public Inventory() : this(0, 0, 0, 0) { }
-
-    //TEMP

```

Comment:
Theres a seperate issue to remove this when were ready to but at the moment michael needs it i think